src/presentational/Checkbox.jsx

```diff
@@ -0,0 +1,47 @@
+import { Component, PropTypes } from 'react';
+
+export default class Checkbox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 'selected',
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(e) {
+    this.props.onChange(e);
+    this.setState({selected: e.target.checked});
+  }
+
+  render() {
+    const options = this.props.options.map((option) =>
+      <div>
+        <input
+          key={option.value}
+          name={this.props.name}
+          type="checkbox"
+          value={option.label}
+          onChange={this.handleChange} />
+            {option.label}
+        <br />
+      </div>
+    )
+    return (
+      <div>
```

Comment:
I'd get rid of the nested `div` elements and mark this up as a `ul` list instead, with each input inside an `li`, then style the list with CSS to get either a vertical or inline layout.